### ISSUE TITLE
[Summon Urn + Bolt] combo / targeting

### ISSUE THREAD

**Issue by SoulMuncher** [jdoleary/Spellmasons#993]
It's very sad that this combo does not work

**Comment by jdoleary** [comment]
Yeah but then you couldn't chain off of urns to use blessings.  I'm not a fan of this exception but i see how it could be frustrating in that case.  Maybe bolt avoids chaining to allies entirely?
That actually could work much better, even if you target an ally it will always chain to enemies.
